dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -137,6 +140,14 @@ export const labelStyles = css`
 	}
 `;
 
+const darkLabelStyles = css`
+	.ad-slot[data-label-show='true']:not(.ad-slot--interscroller)::before {
+		background-color: transparent;
+		border-top-color: ${palette.neutral[20]};
+		color: ${palette.neutral[86]};
```

Comment:
We should probably use `--ad-labels-text` and add explicit colours for audio/video design